c2corg_ui/static/img/logo2.svg

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I already try to do it , i can change the color with css only if the svg is in the html. With a img tag, we cannot.